poleno/utils/admin.py

```diff
@@ -37,3 +37,17 @@ def admin_obj_format(obj, format=u'{tag}', *args, **kwargs):
         except NoReverseMatch:
             pass
     return res
+
+class ReadOnlyAdminInlineMixin(object):
+
+    def get_readonly_fields(self, request, obj=None):
+        return self.fields
+
+    def has_change_permission(self, request, obj=None):
+        return True if request.resolver_match.url_name.endswith(u'change') else False
```

Comment:
@martinmacko47 Pridame parameneter pre `url_name` kde sa v admine edituje instancia. Cize nahradit `u'change'` paramentrom. Neviem ci je vzdy pravidlom ze `url_name` je v tvare `u'<appname>_<modelname>_change'`.

Reply:
URL v admine su vzdy taketo: https://docs.djangoproject.com/en/1.8/ref/contrib/admin/#reversing-admin-urls

Takze `.endswith(u'change')` je v pricime ok. Aj ked trochu hack.

Keby sme to chceli spravit elegantnejsie. Mozno zo `self`, tj. z admin inline instancie, sa da nejak ziskat instancia samotneho adminu. Z neho potom `app_label` a `model_name` a nazov URL vyskladat. Resp. mozno priamo v admine samotnom je nejaka metoda, ktora vrati nazov "change" URL. Skus popatrat, ci sa ti to nejak nepodari. Ak ano, tak to sprav elegantnejsie, zisti nazov "change" URL a porovnaj cele `request.resolver_match.url_name` s tym nazvom "change" URL. Ak sa to nepodari, tak aj sucastne riesenie je good enough.

Reply:
:ok: Ziskal som instanciu parent modelu a z toho som tiez ziskal potrebne udaje `app_label` a `app_name`